localflavor/np/np_districts.py

```diff
@@ -0,0 +1,149 @@
+"""
+Zone wise list of districts of Nepal.
+
+Source: https://en.wikipedia.org/wiki/List_of_zones_of_Nepal
+
+Choices for districts are in this format:
+
+    (('name_of_district'), _('Name of district')),
+"""
+
+from django.utils.translation import gettext_lazy as _
+
+# list of districts in Bagmati Zone
+BAGMATI_DISTRICTS = [ 
+    (('bhaktapur'), _('Bhaktapur')),
```

Comment:
The extra set of brackets aren't needed. It should look like this:
```suggestion
    ('bhaktapur', _('Bhaktapur')),
```
This applies to other places as well.